src/pages/Login/Login.jsx

```diff
@@ -102,9 +103,8 @@ class Login extends Component {
 
     this.setState({ loading: false });
 
-    const TokenGen = ls.get('token');
-
-    if (TokenGen !== 'undefined') {
+    const Token = ls.get('token');
+    if (Token !== 'undefined') {
```

Comment:
Please use camelCase for variable declaration
```suggestion
    const Token = ls.get('token');
    if (Token !== 'undefined') {
```